web/src/app/schedules/calendar-subscribe/CalendarSubscribeForm.js

```diff
@@ -49,7 +20,7 @@ export default function CalendarSubscribeForm(props) {
             fullWidth
             component={TextField}
             name='name'
-            label='Subscription Name'
+            label='Subscription Source Name'
```

Comment:
👍 This should a good way to distinguish that this is for GoAlert organization, and not what they will see on their devices, I think 🎉